modules/cloudaccess/wf_privatelink_manager_definition.json

```diff
@@ -0,0 +1,10 @@
+{
+    "actions": [
+        "Microsoft.Authorization/roleAssignments/read",
+        "Microsoft.Authorization/roleDefinitions/read",
+        "Microsoft.Network/privateEndpoints/*",
+        "Microsoft.Network/networkInterfaces/*",
+        "Microsoft.Network/privateDnsZones/*",
+        "Microsoft.Network/virtualNetworks/*"
```

Comment:
does this actually need * for this on the virtual networks? that feels like a very wide-ranging permission for what this does...